test/src/e2e.long/invalidBlockPropagation.helper.ts

```diff
@@ -116,7 +119,6 @@ async function testBody(
         textraData?: Buffer;
         ttransactionRoot?: H256;
         tstateRoot?: H256;
-        tnextValidatorSetHash?: H256;
```

Comment:
Why did you remove this line?

Reply:
There are other code parts that is removing "tnextValidatorSetHash".
Please review your PR again before request review.

Reply:
This code is one of them: https://github.com/CodeChain-io/foundry/pull/176/files#r380007277